docs/issue--pr-labelling-project-sync-automation-workflows-v1.md

```
@@ -0,0 +1,299 @@
+# **GitHub Labelling Workflows** 
+
+## *Issue types & labelling, PR labelling and Project sync to add missing project meta*
+
+***Version:*** 1.0 • ***Last updated:*** 9 Oct 2025
+
+Below are ready‑to‑drop workflow files for **Issue & PR Labelling** and **Project Sync (add missing project meta)**. Replace placeholders noted with `<>` and set the secrets/variables as documented beneath each file.
+
+---
+
+[.github/workflows/labels-issues-prs.yml](#.github/workflows/labels-issues-prs.yml)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)